src/js/main.js

```diff
@@ -1,3 +1,4 @@
+import "../scss/components/select/multi-select.ts";
```

Comment:
Please could you adjust the import so it matches the others (i.e. starts with `./components/...`)?